netlify/build

```diff
@@ -41,10 +43,19 @@ if [[ "$CONTEXT" = "production" ]]; then
 fi;
 
 # Set up Vale, a text linter
-wget -O- https://github.com/errata-ai/vale/releases/download/v1.2.7/vale_1.2.7_Linux_64-bit.tar.gz | tar -xz vale
+wget --quiet -O- https://github.com/errata-ai/vale/releases/download/v1.2.7/vale_1.2.7_Linux_64-bit.tar.gz | tar -xz vale >/dev/null 2>/dev/null
```

Comment:
The latest Vale is v2.16. Should we try to upgrade?

Reply:
Done.